web/modules/custom/geolayer/modules/map/js/farmOS.behaviors.geolayers.js

```diff
@@ -0,0 +1,9 @@
+(function (Drupal) {
+    farmOS.map.behaviors.farm_nfa_geolayers_map_activities = {
+      attach: function (instance) {
+        console.log('farmOS.map.behaviors.farm_nfa_map_activities.attach')
```

Comment:
I'm guessing this is WIP

Reply:
@pcambra all of these issues have been tackled now, i was working on some refactoring and blockers for this PR, due to which these got missed from getting updated, but all of these have been taken care of now